lxd/daemon.go

```diff
@@ -1078,17 +1078,17 @@ func (d *Daemon) Stop() error {
 	forceStop := false
 
 	d.tomb.Kill(errStop)
-	shared.LogInfof("Stopping REST API handler:")
+	shared.LogDebugf("Stopping REST API handler:")
```

Comment:
Similar to the ones at startup time, the teardown (Stop()) messages should stay at Info level.